x-pack/plugin/src/test/resources/rest-api-spec/api/sql.clear_cursor.json

```diff
@@ -1,6 +1,7 @@
 {
   "sql.clear_cursor": {
     "documentation": "Clear SQL cursor",
+    "stability" : "beta",
```

Comment:
SQL APIs should be GA as of 6.7. @elastic/es-sql can you confirm?

Reply:
Indeed, SQL is GA since 6.7. Thanks @jaymode for picking it up.

Reply:
Updated, to reflect latest state changes. Thanks for confirming.